src/js/modules/infragistics.ui.editors.js

```diff
@@ -7736,6 +7747,9 @@
 
 			// RegEx for /Date(milisecond)/
 			this._mvcDateRegex = /^\/Date\((.*?)\)\/$/i;
+			if (offset !== null && (offset > 840 || offset < -720)) {
+				console.log($.ig.Editor.locale.dateEditorOffsetRange);
```

Comment:
Fair enough, I'll change this case and I've added #681 for other instances to be changed to follow what the majority of controls do with incorrect options.